Documentation/helm-operator.md

```diff
@@ -83,20 +95,21 @@ The following tables lists the configurable parameters of the rook-operator char
 | `resources`        | Pod resource requests & limits       | `{}`                 |
```

Comment:
I think we need to update this configuration sections. We are missing some parameters like Flexvolume_dir, agent toleration and agent toleration keys. It was added in this PR: https://github.com/rook/rook/pull/1402

Reply:
I just realized that the `AGENT_TOLERATION_KEY` feature added in that PR was not added to the helm config. Can you open up a PR to add this?

Reply:
ok, i'll check against the latest operator.yaml